qa/rpc-tests/wallet_mergetoaddress_2.py

```diff
@@ -0,0 +1,341 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal, initialize_chain_clean, \
+    start_node, connect_nodes_bi, wait_and_assert_operationid_status
+import os
+import zipfile
+
+class WalletMergeToAddressTest (BitcoinTestFramework):
+
+    op_result_k = "op_result"
+    balance_delta_k = "balance_delta"
+    joinsplit_len_k = "joinsplit_len"
+    mergingUTXOs_k = "mergingUTXOs"
+    mergingNotes_k = "mergingNotes"
+    mergingTransparentValue_k = "mergingTransparentValue"
+    mergingShieldedValue_k = "mergingShieldedValue"
+
+    def import_data_to_data_dir(self):
```

Comment:
I think it would be nice to add, either here or to the `run_test`, a description of the state being imported, e.g. in terms of chains height, num of utxos for each node and so...

Also, using a more specific name, compared to `test_setup_.zip`, might be more future proof in case we want to implement a similar approach in other tests.

Reply:
Done